homeassistant/components/roborock/strings.json

```diff
@@ -62,6 +62,39 @@
           "custom": "Custom"
         }
       }
+    },
+    "vacuum": {
+      "roborock": {
+        "state_attributes": {
+          "status": {
+            "state": {
```

Comment:
This should be its own sensor, and not a state attribute IMHO.

Reply:
This is how the mioo integration does it, so I figured I'd keep it consistent for users who migrated. But I'm honestly fine with either approach. 

This way is nice as the status state attribute already existed and describes the same thing as the status I have here. 

I'm fine with changing to a sensor if that is what you would prefer though. Just let me know

Reply:
> This is how the mioo integration does it

Sure, but that integration is also pretty old :)

> I'm fine with changing to a sensor if that is what you would prefer though. Just let me know

Yes, that is the preferred way nowadays. This really suits a sensor entity with the `enum` device class.

../Frenck

Reply:
Okay, I will change it then!

Thanks for the feedback!

Reply:
Fixed!